logging/slogging/log.go

```diff
@@ -0,0 +1,37 @@
+// Package slogging provides some useful helpers over the standard `log/slog` package for structured logging.
+package slogging
```

Comment:
J'hésite sur le nom du package.

Sur un autre projet je l'ai juste appelé `logger` pour que ce soit _short and sweet_, mais là vu qu'il y a déjà un package `logging`…

Je veux bien vos avis

Reply:
logger j'aime bien

Reply:
ou alors `log` tout court. Faudra juste pas importer le package de la stdlib 😄

Reply:
> ou alors `log` tout court. Faudra juste pas importer le package de la stdlib 😄

oui c'est pour ça que je préfère éviter